pkg/abstract.go

```diff
@@ -227,6 +229,10 @@ func scrapeDiscoveryJobUsingMetricData(
 		level.Error(logger).Log("msg", "Couldn't get account Id", "err", err)
 		return
 	}
+	if len(resources) == 0 {
+		level.Info(logger).Log("msg", "No tagged resources found")
```

Comment:
```suggestion
		level.Info(logger).Log("msg", "No tagged resources made it through filtering")
```
Thinking about a way to differentiate between this log message and this one http://github.com/grafana/yet-another-cloudwatch-exporter/blob/69f62865ed5f593ceb9629ff7f91c10b42f57865/pkg/aws_tags.go#L140. The linked one is more of an "error" while this one might not be a problem due to other filters being applied after the raw listing. It's unfortunate that listing resources is tightly coupled to filtering resources like this.